sdk/v1/client_test.go

```diff
@@ -43,7 +43,21 @@ var (
 		},
 		NumberOfDocuments: 10,
 		TaskName:          "zebedee",
-		ETag:              `"5feferwgg44rggsdbrr54lklnhssss"`,
+	}
+
+	expectedJob = models.Job{
+		ID:          "883c81fd-726d-4ea3-9db8-7e7c781a01cc",
```

Comment:
It's best to make use of the variable here:
```suggestion
		ID:          testJobID,
```